### PR TITLE
Log ring in stress tests runs

### PR DESCRIPTION
**Pull request by RishhiB**
What: This ticket changes our encoding system, specifically the FRS endpoint and rings. The goal is to establish a more structured encoding system that separates services and rings.

Why:

1. Scalability: It is likely that new rings will be added, we dont have a separation of services and rings yet.
2. Telemetry: We can log based on ring without convoluted kusto queries.

Acceptance Criteria:

1. The new encoding system should distinguish between service and rings.
2. The system should be capable of adding new rings with minimal additional coding.
3. Telemetry should provide a logs of each service and ring without convoluted kusto queries.
https://github.com/microsoft/FluidFramework/pull/15054
[AB#4187](https://dev.azure.com/fluidframework/235294da-091d-4c29-84fc-cdfc3d90890b/_workitems/edit/4187)

### CODE REVIEW

**Comment by vladsud** [comment]
There is no need to create anything in Kusto - columns are auto-created once events start flowing.
But clearly this PR is not enough, as proper implementation should get rid of frsCanary term from repo. The way it's structured right now will break existing flow.

File: packages/test/test-driver-definitions/package.json [hunk 1]
```diff
@@ -82,13 +82,6 @@
...
 	"typeValidation": {
-		"broken": {
-			"TypeAliasDeclaration_DriverEndpoint": {
-				"backCompat": false
-			},
-			"TypeAliasDeclaration_RouterliciousEndpoint": {
-				"backCompat": false
-			}
-		}
+		"broken": {}
```
Comment (alexvy86) [inline packages/test/test-driver-definitions/package.json:85]: This change seems like it shouldn't be here? We only clean up this section as part of the release process.

File: packages/test/test-service-load/src/runner.ts [hunk 2]
```diff
@@ -96,6 +96,7 @@ async function main() {
 		driverType: driver,
 		driverEndpointName: endpoint,
 		profile: profileName,
+		ring: endpoint,
```
Comment (alexvy86) [inline packages/test/test-service-load/src/runner.ts:99]: This doesn't feel right. When `orchestratorProcess` from nodeStressTest.ts spawns `dist/runner.js`, it probably should pass `--ring` as a parameter to it as well, so we have the value here.

File: packages/test/test-service-load/src/utils.ts [hunk 3]
```diff
@@ -177,6 +178,7 @@ export async function initialize(
 		driverEndpointName: testDriver.endpointName,
 		profile: profileName,
 		runId: undefined,
+		ring: undefined,
```
[the remark this answers is not in the document]
Comment (anthony-murphy) [reply packages/test/test-service-load/src/utils.ts:181]: to make a change like this it should push all the way down to the test driver, and go everywhere driver type and endpoint name are currently used.

File: packages/test/test-driver-definitions/package.json (hunk 1 above)
Comment (RishhiB) [reply packages/test/test-driver-definitions/package.json:85]: Hey Alejandro! I was just reverting the changes I made in the parent PR. I added this in when I added in frsCanary to RouterliciousEndpoint; https://github.com/microsoft/FluidFramework/pull/15054/files. 
we were removing the frsCanary option from RouterliciousEndpoint which causes build errors unless I remove this.

File: packages/test/test-service-load/src/utils.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (RishhiB) [reply packages/test/test-service-load/src/utils.ts]: Created tasks AB#4491 and AB#4490 for improvements that are out of the scope of this PR

**Pull request closed without merging**